examples/realworld/ext/user/actions.js

```diff
@@ -0,0 +1,50 @@
+import HttpError from '@wasp/core/HttpError.js'
+
+export const signup = async ({ username, email, password }, context) => {
```

Comment:
You can use our cool new signup function: https://wasp-lang.dev/docs/language/basic-elements#signup

Reply:
Awesome, used it now!